packages/server/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "server",
   "version": "1.0.0",
   "scripts": {
-    "start": "node src/index.ts",
+    "start": "nodemon --inspect src/index.ts",
```

Comment:
Is this just for debug-ability? What value does it add?

Reply:
The "nodemon" makes it reload every time that you make a change to the file.
The "--inspect" makes it possible to connect a debugger to it.  You don't *have* to attach, but it just establishes the web socket for it.  See more in the additions that I made to the contributing doc.